tests/Psl/Iter/ReduceKeysTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Tests\Iter;
+
+use PHPUnit\Framework\TestCase;
+use Psl\Iter;
+
+class ReduceKeysTest extends TestCase
```

Comment:
```suggestion
final class ReduceKeysTest extends TestCase
```

I will probably do a PR tomorrow to switch all test cases to final. it's annoying to see them in the suggestions when typing `Foo extends |` 😅